Bottom: f98cf7145f298b81b5e34cf5e7ed2d84f4a5b98d
Top:    c08b6b11bce33e1f847081c87863469569743b94
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-25 09:27:56 -0500

Refresh of update-gitlab-to-10-1

---
diff --git a/ansible-playbook/library/docker_volume_clone.py b/ansible-playbook/library/docker_volume_clone.py
index 34611e26..b9a360e7 100644
--- a/ansible-playbook/library/docker_volume_clone.py
+++ b/ansible-playbook/library/docker_volume_clone.py
@@ -1,12 +1,24 @@
 #!/usr/bin/python
 #
 # Copyright 2016 Red Hat | Ansible
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# GNU General Public License v3.0+ (see COPYING
+# or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
 
+try:
+    import docker
+    from docker import utils
+except ImportError:
+    # missing docker-py handled in docker_common
+    pass
+finally:
+    from ansible.module_utils.docker_common import (AnsibleDockerClient,
+                                                    DockerBaseClass)
+
+
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
@@ -63,15 +75,6 @@ EXAMPLES = '''
 RETURN = '''
 '''
 
-try:
-    import docker
-    from docker import utils
-except ImportError:
-    # missing docker-py handled in docker_common
-    pass
-
-from ansible.module_utils.docker_common import AnsibleDockerClient, DockerBaseClass
-
 
 class VolumeManager(DockerBaseClass):
 
@@ -103,7 +106,8 @@ class VolumeManager(DockerBaseClass):
         )
 
     def clone_volume(self):
-        cmd = "apk add --no-cache rsync && /usr/bin/rsync --archive --del /from/ /to/"
+        cmd = """apk add --no-cache rsync \
+                 && /usr/bin/rsync --archive --del /from/ /to/"""
         stdout = ''
         try:
             stdout = self.hlclient.containers.run(
